modules/social_features/social_post/src/Entity/Post.php

```diff
@@ -157,6 +157,58 @@ public function getDisplayName() {
     return t('post');
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function getVisibility() {
+    if ($this->hasField('field_visibility')) {
+      switch ($this->field_visibility->value) {
+
+        case "0":
```

Comment:
You can remove `case "2"` at line 175-177.
Just combine it like:

```suggestion
        case "0":
        case "2":
```